core/src/main/java/com/microsoft/applicationinsights/internal/channel/common/TransmissionNetworkOutput.java

```diff
@@ -185,6 +187,12 @@ public boolean sendSync(Transmission transmission) {
             } catch (IOException ioe) {
                 ex = ioe;
                 logger.error("Failed to send", ioe);
+            } catch (FriendlyException e) {
+                ex = e;
+                if(!friendlyExceptionThrown) {
+                    logger.error(e.getMessage());
```

Comment:
let's log at the bottom of the message something like

`(this message is only logged the first time it occurs after startup)`